explainer.html

```diff
@@ -271,56 +229,36 @@ <h2>Use Cases and Requirements</h2>
-        <br>
-        Requirement: Lossless conversion between highly compact and less compact forms of informationally identical RDF Datasets.
+        <a href="https://www.w3.org/TR/rdf11-concepts/#section-skolemization">Skolem IRIs</a> have been proposed in RDF 1.1 as a way to replace blank nodes with IRIs in application scenarios where it is preferable to avoid the use of blank nodes. Rather than using an ad hoc scheme to generate Skolem IRIs to replace blank nodes, an alternative is to generate Skolem IRIs in a deterministic manner, such that compliant implementations will generate the same IRIs to replace the same blank nodes in isomorphic copies of an RDF graph or dataset. Such a procedure will produce a canonical version of a Skolemized RDF graph or dataset that can then be used in the context of several of the use-cases mentioned previously.
       </dd>
 
       <dt id="semanticConsistency">Semantic consistency of multi-part datasets</dt>
-      <dd>The <a href="#detectingChanges">detecting changes</a> and <a href="#spaceEfficient">space-efficient verification</a> use-cases above can be leveraged in situations where a graph or dataset semantically relies on one or several other graph(s), which it refers to through links. Attaching cryptographic hashes to these links would allow to verify the overall integrity of the set of interconnected graphs. One such example is the <a href="https://www.w3.org/TR/owl2-primer/#a_imports">import mechanism of OWL</a>: the ontology consumer may wish to verify that the imported ontology is the same as the one used by the author of the importing ontology, otherwise the resulting inferences may differ. Another such example are <a href="https://www.w3.org/TR/EARL10-Schema/">EARL test reports</a>: the consumer may wish to ensure that the test description pointed to by the report is the one that was actually used for the test.
-        <br>
-        Requirement: A standard way of computing and attaching a cryptographic hash to a graph or dataset.
+      <dd>The <a href="#detectingChanges">detecting changes</a> and <a href="#spaceEfficient">space-efficient verification</a> use-cases above can be leveraged in situations where a graph or dataset semantically relies on one or several other graph(s), which it refers to through links. Attaching cryptographic hashes to these links would enable the verification of the overall integrity of the set of interconnected graphs. One such example is the <a href="https://www.w3.org/TR/owl2-primer/#a_imports">import mechanism of OWL</a>: the ontology consumer may wish to verify that the imported ontology is the same as the one used by the author of the importing ontology, otherwise the resulting inferences may differ. Another such example are <a href="https://www.w3.org/TR/EARL10-Schema/">EARL test reports</a>: the consumer may wish to ensure that the test description pointed to by the report is the one that was actually used for the test.
```

Comment:
```suggestion
      <dd>The <a href="#detectingChanges">change detection</a> and <a href="#spaceEfficient">space-efficient verification</a> use-cases above can be leveraged in situations where a graph or dataset semantically relies on one or several other graph(s), which it refers to through links. Attaching cryptographic hashes to these links would enable the verification of the overall integrity of the set of interconnected graphs. One such example is the <a href="https://www.w3.org/TR/owl2-primer/#a_imports">import mechanism of OWL</a>: the ontology consumer may wish to verify that the imported ontology is the same as the one used by the author of the importing ontology, otherwise the resulting inferences may differ. Another such example is an <a href="https://www.w3.org/TR/EARL10-Schema/">EARL test report</a>: the consumer may wish to ensure that the test description pointed to by the report is the one that was actually used for the test.
```